### ISSUE TITLE
Accessibility Critiques

### ISSUE THREAD

**Issue by savannanoh** [amyjko/user-interface-software-and-technology#163]
- *Autism is not an intellectual disability, it is a developmental disorder. Please correct this harmful mistake.*
- Neurodiversity is largely ignored in this chapter +1
- Talk about gaps and needs in innovations 
- Discuss how tech companies view and approach accessibility
- Do truly universally accessible interfaces exist? What are some principles for making a universal design? +5
- What accessibility standards should be met before a product gets shipped?
- Talk about how the Genie system works
- How can designers advocate for accessibility?
- Why is it difficult to design for accessibility in industry?
- Go over the history of accessibility and technology
- Talk about how accessibility can be considered at various stages in product development
- Talk about accessibility regulation in commercial industries
- Talk about side effects that might limit usability to certain groups by designing for other groups and how this might affect people with multiple disabilities +1
- The chapter talks about adopting GUIs because of vision as a strong sense, but could it also be due to the ease of technologically implementing GUIs versus other types of interfaces?
- How do the people who need it actually get access to accessible technologies?
- Who teaches people how to use these accessible technologies?
- Why does this chapter cite a World Bank survey from 2004? Seems old
- Talk more about the effects of UI as exclusion
- explain why the current model of user interface is still used, and developers still use the same toolkits to make interfaces, even though they know about its exclusivity
- Include key actionable points for designing for accessibility
- Talk about the metaverse and accessibility and other UIs, like watches, earphones, headsets. Talk about assistive technology using different types of hardware, such as IoT, ubicomp, smart home devices +2
- Talk about advances in computer vision and how this can help people with visual impairments
- Talk about tradeoffs/tensions between accessibility and privacy +1
- Why are accessibility toolkits not more integrated into products/software? Why is the industry hesitant?
- Have concrete examples of how designing for accessibility is designing for all
- Talk about heuristic evaluations in this chapter
- Talk about neuroscience and how BCIs could help eliminate hand movements and other barriers altogether
- Have markers/headings for different disabilities
- Which is better–access technologies or universal interfaces? Should we aim to design universal interfaces or design interfaces that can effectively and seamlessly implement access technologies?
- How might these workaround technologies prevent progress towards universal accessibility?
- Grammar/typos:
  - "Other _works hows_ *work shows* that emoji"
  - Some work shows that some of _the the_ information needs in these descriptions are universal
  - Broken link to Section 508
-

**Comment by savannanoh** [comment]
World Bank stat is from 18 years ago. Student linked a more recent article (https://www.upi.com/Health_News/2018/08/16/CDC-1-in-4-adults-have-a-disability-that-impacts-daily-life/8421534423136/) that states that 1 in 4 adults have a disability that impacts daily life.

**Comment by savannanoh** [comment]
Chapter doesn't define machine vision
